docs/HowTo/Configure/Use-Configuration-File.md

```diff
@@ -0,0 +1,63 @@
+---
+description: Use Teku configuration file
+---
+
+# Using the Teku configuration file
+
+To specify command line options in a file, use a YAML configuration file.
+
+Save the configuration file and reuse it across node startups. To specify the configuration file,
+use the [`--config-file`](../../Reference/CLI/CLI-Syntax.md#config-file) option.
+
+To override an option specified in the configuration file, either specify the same option on the
+command line or as an
+[environment variable](../../Reference/CLI/CLI-Syntax.md#teku-environment-variables). For options
+specified in more than one place, the order of precedence is command line, environment variable,
+configuration file.
+
+## YAML specification
+
+The configuration file must be a valid YAML file composed of key/value pairs. Each key is the same
+as the corresponding command line option name without the leading dashes (`--`).
```

Comment:
could simplify to - Each key is the corresponding command line option without the leading...

Reply:
Done